spec/lib/va_profile/profile/v3/health_benefit_bio_response_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'va_profile/profile/v3/health_benefit_bio_response'
+
+describe VAProfile::Profile::V3::HealthBenefitBioResponse do
+  subject { described_class.new(response) }
+
+  let(:response) do
+    double('Faraday::Response',
+           status: 200,
+           body: {
+             'profile' => {
+               'health_benefit' => {
+                 'associated_persons' => [{
+                   'contact_type' => contact_type
+                 }]
+               }
+             }
+           })
+  end
+
+  describe 'Emergency contact' do
+    let(:contact_type) { 'Emergency Contact' }
```

Comment:
I suggest using the value from VAProfile::Models::AssociatedPerson instead of hard coding it here and below.

Reply:
Waiting for this comment to be addressed

Reply:
This was intentional. If values change in the code, these tests will fail.